io.github.adrienverge.PhotoCollage.yml

```diff
@@ -0,0 +1,25 @@
+app-id: io.github.adrienverge.PhotoCollage
+runtime: org.gnome.Platform
+runtime-version: "44"
+sdk: org.gnome.Sdk
+rename-icon: photocollage
+rename-appdata-file: photocollage.appdata.xml
+rename-desktop-file: photocollage.desktop
+command: photocollage
+finish-args:
+  - --share=ipc
+  - --device=dri # hardware acceleration
+  - --filesystem=home
+  - --socket=fallback-x11
+  - --socket=wayland
+
+modules:
+  - python3-Pillow.json
+  - name: photocollage
+    buildsystem: simple
+    build-commands:
+      - pip3 install --prefix /app --no-deps .
+    sources:
+      - type: git
+        url: https://github.com/adrienverge/PhotoCollage.git
+        branch: master
```

Comment:
this should be a commit or a tag. NEVER a branch.

Reply:
Alright, good to know! I changed it now